src/patternfly/components/Nav/examples/Navigation.md

```diff
@@ -135,7 +135,7 @@ import './Navigation.css'
 ```hbs
 {{#> nav nav--attribute='aria-label="Global"'}}
   {{#> nav-list}}
-    {{#> nav-item nav-item--IsExpandable="true" nav-item--expanded="true" nav-item--current="true"}}
+    {{#> nav-item nav-item--IsExpandable="true" nav-item--IsExpanded="true" nav-item--current="true"}}
```

Comment:
```suggestion
    {{#> nav-item nav-item--IsExpandable=true nav-item--IsExpanded=true nav-item--current=true}}
```

Technically these settings are [boolean values](https://handlebarsjs.com/guide/expressions.html#helpers) rather than strings. I don't believe we need to change all instances, but it would be nice to omit quotes as we update code.